Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM python:3.7-alpine
 
 ENV ZOO_RDS_PRODUCTION_CERT_PATH=/etc/ssl/certs/rds-combined-ca-bundle.pem \
```

Comment:
We can also remove this one now that we are here. It can be set up in the final image

Reply:
yes, true

Reply:
The problem is that I would have to parse the CERT PATH from the DATABASE_OPTIONS, so will we download the CERT also in the final image and remove both variables?

Reply:
Yep, those variables should be set up on the final deployment. This should work without SSL mode in the DB conn, and we'll add it on the final image.

Reply:
So remove the download of the cert as well